lckernel/cad/storage/settings/modulesettings.cpp

```diff
@@ -19,7 +19,7 @@ ModuleSettings::~ModuleSettings() {
 }
 
 const ISettingValue* ModuleSettings::get(const std::string& name) const {
-    std::cout << name << std::endl;
+   // std::cout << name << std::endl;
```

Comment:
Don't keep commented code, decomment it or delete it

Reply:
Actually it was not commented and was printing on terminal the settings so i commented it. Should i still delete it?